voxpupuli-puppet-lint-plugins.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'puppet-lint-file_ensure-check', '~> 2.0'
   s.add_dependency 'puppet-lint-leading_zero-check', '~> 2.0'
   s.add_dependency 'puppet-lint-lookup_in_parameter-check', '~> 2.0'
-  s.add_dependency 'puppet-lint-manifest_whitespace-check', '~> 0.3', '< 1.0.0'
+  s.add_dependency 'puppet-lint-manifest_whitespace-check', '~> 1.0'
```

Comment:
Than **require** puppet-lint-manifest_whitespace-check 1.x, and does not match the PR title :smile: